front/src/routes/settings/settings-system/style.css

```
@@ -1,3 +1,12 @@
-.textDecorationNone {
-  text-decoration: none !important;
+.spin {
+  animation: settingsSystemSpin 1s linear infinite;
+}
+
+@keyframes settingsSystemSpin {
```

Stylelint (17.14.1)

[error] Expected keyframe name "settingsSystemSpin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)